src/request/src/middleware/src/storage/index.js

```diff
@@ -14,14 +14,14 @@ export {
   MemoryAdapter
 };
 
-export default class Storage {
+export class Storage {
```

Comment:
In this case, any consumer of the `Storage` class should only interface with the `Storage` class and not any of the storage adapters. By using `index.js` for the `Storage` class, import statements look like `import { Storage } form './storage';` If it was separated into a different file and then exported with an `index.js` file the effect would be the same. This removes that redundancy.